README.md

```diff
@@ -77,38 +78,73 @@ This sample app lives in this repo in the `sample` directory.
 
 ## Jetpack Compose support
 
-Radiography will automatically render Composables found in your view tree if the Compose Tooling
-library is on the classpath. If you're using Compose, this is likely already the case (the
+[Jetpack Compose](https://developer.android.com/jetpack/compose) is Google's new UI toolkit. It is
+currently in pre-release status, and changes frequently. However, if you're already experimenting
+with Compose in your apps, any UI that is defined in Compose will not show up as Android views.
+
+Radiography will automatically render composables found in your view tree if the Compose Tooling
+library is on the classpath. If you are using Compose, this is likely already the case (the
 `@Preview` annotation lives in the Tooling library). On the other hand, if you're not using Compose,
 Radiography won't bloat your app with transitive dependencies on any Compose artifacts.
 
 Compose releases frequently. If you are using Radiography with an unsupported version of Compose,
-or you don't depend on the Tooling library, then Radiography will include a message in the result
-asking you to upgrade Radiography or add the Tooling library, but this is optional.
+or you don't depend on the Tooling library, then Radiography will still try to detect compositions,
+but instead of rendering the actual hierarchy, it will just include a message asking you to upgrade
+Radiography or add the Tooling library.
 
 ### Compose usage
 
-Compose support is experimental right now, since Compose is still changing a lot on a regular basis.
+_Compose support is experimental right now, since Compose is still changing a lot on a regular
```

Comment:
> changing a lot on a regular basis.

=>

> changing frequently.

?

Reply:
Going to remove this sentence from this paragraph and move it to the one above.

Reply:
Done.